modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -17,16 +17,40 @@
 
 package org.apache.ignite.internal.raft;
 
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.ignite.network.ClusterNode;
 import org.apache.ignite.network.ClusterService;
+import org.apache.ignite.raft.client.Peer;
+import org.apache.ignite.raft.client.message.RaftClientMessageFactory;
+import org.apache.ignite.raft.client.message.impl.RaftClientMessageFactoryImpl;
+import org.apache.ignite.raft.client.service.RaftGroupCommandListener;
+import org.apache.ignite.raft.client.service.RaftGroupService;
+import org.apache.ignite.raft.client.service.impl.RaftGroupServiceImpl;
+import org.apache.ignite.raft.server.RaftServer;
+import org.apache.ignite.raft.server.impl.RaftServerImpl;
 
 /**
  * Best raft manager ever since 1982.
  */
 // TODO: IGNITE-14586 Remove @SuppressWarnings when implementation provided.
```

Comment:
Cause you've removed @SuppressWarnings please also remove TODO: IGNITE-14586

Reply:
Done.